plugins/integrations/kubernetes-service/src/main/java/org/apache/cloudstack/api/command/user/kubernetes/cluster/ScaleKubernetesClusterCmd.java

```diff
@@ -58,19 +61,38 @@
     //////////////// API parameters /////////////////////
     /////////////////////////////////////////////////////
     @Parameter(name = ApiConstants.ID, type = CommandType.UUID, required = true,
-            entityType = KubernetesClusterResponse.class,
-            description = "the ID of the Kubernetes cluster")
+        entityType = KubernetesClusterResponse.class,
+        description = "the ID of the Kubernetes cluster")
     private Long id;
 
     @ACL(accessType = SecurityChecker.AccessType.UseEntry)
     @Parameter(name = ApiConstants.SERVICE_OFFERING_ID, type = CommandType.UUID, entityType = ServiceOfferingResponse.class,
-            description = "the ID of the service offering for the virtual machines in the cluster.")
+        description = "the ID of the service offering for the virtual machines in the cluster.")
     private Long serviceOfferingId;
 
     @Parameter(name=ApiConstants.SIZE, type = CommandType.LONG,
-            description = "number of Kubernetes cluster nodes")
+        description = "number of Kubernetes cluster nodes")
     private Long clusterSize;
 
+    @Parameter(name = ApiConstants.NODE_IDS,
+        type = CommandType.LIST,
+        collectionType = CommandType.UUID,
+        entityType = UserVmResponse.class,
+        description = "the IDs of the nodes to be removed")
+    private List<Long> nodeIds;
+
+    @Parameter(name=ApiConstants.AUTOSCALING_ENABLED, type = CommandType.BOOLEAN,
```

Comment:
We don't support allowing autoscaling while deployment itself and it can be enabled only after the cluster is deployed?

Reply:
Yes, for now only after deployment, however, I do plan to add that in the future